package.json

```diff
@@ -67,5 +67,5 @@
       "last 1 safari version"
     ]
   },
-  "proxy": "http://localhost:8000"
+  "proxy": "http://localhost:8010"
```

Comment:
The KVStore runs locally at this port.

Reply:
All the stuff  runs over 8010? or we need to switch depends the service? Is not better used something like this:
```
"proxy": {
  "/api": {
    "target": "http://localhost:8000/"
    },
  "/kvstore": {
    "target": "http://localhost:8010/"
  }
}
```

Reply:
Unfortunately, you can just add additional endpoints into the `package.json` but I added the `http-proxy-middleware` package so we can customize proxies by endpoint.